src/components/Home.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './style/Home.css';
+import { Link } from 'react-router-dom';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import europe from '../assets/europe.svg';
+import afirca from '../assets/africa.svg';
+import asia from '../assets/asia.svg';
+import america from '../assets/america.svg';
+import oceania from '../assets/oceania.svg';
+import antarctic from '../assets/antarctica.svg';
+
+const continentList = [
+  { id: 1, continent: 'Africa', map: afirca },
+  { id: 2, continent: 'America', map: america },
+  { id: 3, continent: 'Antarctic', map: antarctic },
+  { id: 4, continent: 'Asia', map: asia },
+  { id: 5, continent: 'Europe', map: europe },
+  { id: 6, continent: 'Oceania', map: oceania },
+];
+
+export default function Home() {
```

Comment:
- [ ] Your Homepage should have a search or filtering feature that allows users to filter items on the homepage. Kindly implement that.